README.md

```diff
@@ -22,9 +22,28 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
 
 ### 2. Install Dependencies
 
+Superuser permissions are necessary to install the Topical Guide as shown in the current instructions.
```

Comment:
Since we're using python3 now, I'd say we make that clear in the readme and
recommend that they use the python3 virtual environment:
https://docs.python.org/3/library/venv.html,
https://docs.python.org/3/tutorial/venv.html.  I vaguely recall there being
a proposal to get rid of pyvenv as a command on the system path, but I
can't seem to find it.

Apparently, you can use `pip3 install <package> --user` to install python
packages locally (so you wouldn't need superuser privileges).  That might
be another noteworthy point to add to the readme.

On Wed, Feb 10, 2016 at 9:13 AM, Connor Cook <notifications@github.com>
wrote:

> In README.md
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87#discussion_r52480160>
> :
>
> > @@ -22,9 +22,28 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
> >
> >  ### 2. Install Dependencies
> >
> > +Superuser permissions are necessary to install the Topical Guide as shown in the current instructions.
>
> Maybe here we should just say something about how we are using virtualenv
> to avoid the need for superuser permissions? It sounds like we are telling
> the user that you actually need superuser instructions to do what is talked
> about below.
>
> Also, do we still use virtualenv? I think Python3 uses pyvenv, or
> something like that, instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87/files#r52480160>.
>



-- 
Nozomu Okuda